services/template.py

```diff
@@ -0,0 +1,28 @@
+from jinja2 import Environment, PackageLoader, StrictUndefined, select_autoescape
+
+_template_service = None
+
+
+def get_template_service():
+    return _get_cached_template_service()
+
+
+def _get_cached_template_service():
```

Comment:
Do we need this to be a singleton?...
I ask because it doesn't seem to be a pattern we use much with `services` in the worker, and I don't know if it provides benefits to implement it like that.

My real concern is that we had issues before with tests (read ATS) around the cached ENV.

I'm not opposed to this idea, I'm just curious what are the benefits of doing it as a singleton. Does the loading process takes long? Is that recommended?

Reply:
I don't think it has to be, I guess it does add complexity for no _real_ reason, i just thought it's probably an object that we don't need to instantiate a bunch of times, it's sort of just like load it and have it stay the same until we redeploy, but you're correct that we don't _need_ it to be a singleton